gammapy/utils/metadata.py

```diff
@@ -230,6 +237,53 @@ class PointingInfoMetaData(MetaData):
     altaz_mean: Optional[AltAzSkyCoordType] = None
 
 
+class TimeInfoMetaData(MetaData):
+    """General metadata information about the time range of an observation.
+
+    Parameters
+    ----------
+    time_start : `~astropy.time.Time` or str
+        The observation start time.
+    time_stop : `~astropy.time.Time` or str
+        The observation stop time.
+    reference_time : `~astropy.time.Time` or str
+        The observation reference time."""
+
+    _tag: ClassVar[Literal["pointing"]] = "time_info"
```

Comment:
The tag is really in the 'pointing' section? Or this is just a copy/paste typo?

Reply:
it is a typo indeed